src/drivers/deploy/service.yaml

```diff
@@ -20,6 +20,7 @@ cluster-type:
 
 prerequisite:
   - cluster-configuration
+  - drivers
```

Comment:
Please remove this.

You can ignore services with `cluster-type: yarn` because they are not supported now.